docs/example-narratives/examples/seasonal-ratings-proposals-slim-patch.json

```diff
@@ -0,0 +1,99 @@
+{
+  "proposal-header": {
+    "source": {
+      "last-updated": "2024-11-20T15:05:43.044267100-07:00",
+      "provider": "UTILITY-A",
+      "origin-id": "5aeacb25-9b65-4738-8a00-ac10afa63640"
+    },
+    "default-emergency-durations": [
+      {
+        "name": "emergency",
+        "duration-minutes": 240
+      }
+    ],
+    "power-system-resources": [
+      {
+        "resource-id": "8badf00d",
+        "alternate-identifiers": [
+          {
+            "name": "segmentX",
+            "authority": "TO-NERC-ID"
+          }
+        ]
+      },
+      {
+        "resource-id": "f34d3d",
+        "alternate-identifiers": [
+          {
+            "name": "segmentY",
+            "authority": "TO-NERC-ID"
+          }
+        ]
+      }
+    ],
+    "default-seasonal-schedule": {
+      "schedule": [
+        {
+          "season-name": "WINTER",
+          "begins": "2024-11-15T00:00:00-05:00"
+        },
+        {
+          "season-name": "SPRING",
+          "begins": "2025-03-01T00:00:00-05:00"
+        },
+        {
+          "season-name": "SUMMER",
+          "begins": "2025-06-15T00:00:00-05:00"
+        },
+        {
+          "season-name": "FALL",
+          "begins": "2025-09-01T00:00:00-05:00"
+        }
+      ],
+      "ends": "2025-11-15T00:00:00-05:00"
```

Comment:
@getorymckeag lmk what you think of this. I believe it addresses the RC-defined seasons use case but still maps to an explicit seasonal rating schedule. We'll need to write down a lot about how the PATCH is interpreted, but I think the server would require that the current date is included in one of the seasons and that at least a full year is covered from the `begins` of that current season through `ends`